lib/app/features/feed/views/pages/article_details_page/article_details_page.dart

```diff
@@ -64,22 +64,27 @@ class ArticleDetailsPage extends HookConsumerWidget {
               slivers: [
                 SliverList(
                   delegate: SliverChildListDelegate([
+                    SizedBox(height: 13.0.s),
                     ScreenSideOffset.small(
-                      child: Padding(
-                        padding: EdgeInsets.only(top: 13.0.s, bottom: 16.0.s),
-                        child: ArticleDetailsDateTopics(
-                          publishedAt: articleEntity.data.publishedAt,
-                          topics: const [
-                            'Technology',
-                            'Crypto',
-                          ], // TODO: get topics from articleEntity
-                        ),
+                      child: ArticleDetailsDateTopics(
+                        publishedAt: articleEntity.data.publishedAt,
+                        topics: const [
+                          'Technology',
+                          'Crypto',
+                        ], // TODO: get topics from articleEntity
```

Comment:
Maybe it's possible to mock topics in articleEntity, to mimic API as much as possible. The goal is to implement everything as much as possible to reduce effort during removing mocks.